jwst/regtest/test_miri_mrs_spec2.py

```diff
@@ -78,6 +78,9 @@ def test_miri_mrs_spec2(run_spec2, fitsdiff_default_kwargs, suffix, rtdata_modul
     rt.is_like_truth(rtdata, fitsdiff_default_kwargs, suffix, truth_path=TRUTH_PATH)
 
 
+# MIRI WCS computations sometimes raise runtime warnings for invalid
+# values.  Ignore the warnings for this test.
+@pytest.mark.filterwarnings("ignore: invalid value")
```

Comment:
Do we want to be a bit more specific?
```suggestion
@pytest.mark.filterwarnings("ignore: invalid value:RuntimeWarning")
```